node/speech-recognition-with-huggingface/src/main.js

```diff
@@ -1,73 +1,47 @@
 import { HfInference } from '@huggingface/inference';
-import { throwIfMissing } from './utils.js';
-import AppwriteService from './appwrite.js';
+import { getStaticFile, throwIfMissing } from './utils.js';
 
-export default async ({ req, res, log, error }) => {
-  throwIfMissing(process.env, [
-    'HUGGING_FACE_API_KEY',
-    'APPWRITE_API_KEY',
-  ]);
+export default async ({ req, res }) => {
+  throwIfMissing(process.env, ['HUGGINGFACE_ACCESS_TOKEN']);
```

Comment:
In the readme this is called `HUGGINGFACE_API_TOKEN`

Reply:
fixed the readme